modules/compute_disk_snapshot/variables.tf

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "name" {
+  description = "Name of the resource policy to create"
+  type        = string
+}
+
+variable "project" {
+  description = "The project ID where the resources will be created"
+  type        = string
+}
+
+variable "region" {
+  description = "Region where resource policy resides"
+  type        = string
+}
+
+variable "snapshot_schedule_policy" {
```

Comment:
We should either make this module for a single policy (preferred), or change this to `snapshot_schedule_policies`.

Reply:
Agree, that was the intent - to make it for a single policy, so the variable name is correct

Reply:
Alright, in that case could we split this into separate variables? Ex. `retention_policy`, `schedule`, and `snapshot_properties`? One huge object is harder to configure.

Reply:
Indeed it makes sense to do so. Pushed the new code